ofl/overpassmono/DESCRIPTION.en_us.html

```diff
@@ -1 +1,12 @@
-<p>An open source font family inspired by Highway Gothic. Sponsored by <a href="https://www.redhat.com/">Red Hat</a> &mdash; Created by <a href="https://www.delvefonts.com">Delve Fonts</a>.</p>
\ No newline at end of file
+<p>Overpass Mono is a thoughtful, monospaced re-imagining of the <a href="https://fonts.google.com/specimen/Overpass">Overpass</a> proportional design, designed by <a href="https://www.delvefonts.com">Delve Fonts</a>. Consisting of five weights ranging from Light to Bold. The Overpass Mono fonts are finely tuned to meet the requirements of today’s programmers.</p>
+<p>The design of Overpass is an interpretation of the well-known “Highway Gothic” letterforms from the Standard Alphabets for Traffic Control Devices published by the U.S. Federal Highway Administration. Starting from those specifications, critical adjustments were made to the letterforms to create an optimal presentation at smaller sizes on-screen and later for display sizes, especially in the lighter weights.</p>
+<p>Overpass can be used for everything from extended passages of text in print and online, to UI design, posters, wayfinding signage, and environmental graphic design.</p>
+
+<ul>
+<li>Designers: Delve Withrington, Dave Bailey, and Thomas Jockin</li>
```

Comment:
Add authors to metadata.pb + link to the other Overpass

Reply:
There's already a link to the other Overpass embedded in the description. Did you want it to be more explicit?